<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="methods" name="Test1(failed)">
    <classes>
      <class name="com.Vtiger.TC.TC002_CreateOrganisationDDTest">
        <methods>
          <include name="loginToApp"/>
          <include name="setUp"/>
          <include name="logoutFromApp"/>
          <include name="disconnectFromDB"/>
          <include name="closeBrowser"/>
          <include name="launcBrowser_URL"/>
          <include name="createOrgWithDD"/>
        </methods>
      </class> <!-- com.Vtiger.TC.TC002_CreateOrganisationDDTest -->
    </classes>
  </test> <!-- Test1(failed) -->
  <test thread-count="5" parallel="classes" name="Test2(failed)">
    <classes>
      <class name="com.Vtiger.TC.TC003_CreateContactWithOrganisationTest">
        <methods>
          <include name="createContactOnly"/>
          <include name="loginToApp"/>
          <include name="setUp"/>
          <include name="logoutFromApp"/>
          <include name="disconnectFromDB"/>
          <include name="closeBrowser"/>
          <include name="launcBrowser_URL"/>
        </methods>
      </class> <!-- com.Vtiger.TC.TC003_CreateContactWithOrganisationTest -->
    </classes>
  </test> <!-- Test2(failed) -->
</suite> <!-- Failed suite [Suite] -->
